README.md

```diff
@@ -263,7 +263,7 @@ class Regressor(ivy.Module):
 
 If we put it all together, we\'ll have something like this. This example
 uses PyTorch as the backend, but this can easily be changed to your
-favorite framework, such as TensorFlow, or JAX.
+favorite frameworks, such as TensorFlow, or JAX.
```

Comment:
I guess we are referring to a singular framework here, not sure if that is right

Reply:
I mean we are referring to multiple of them (TensorFlow, or JAX). That's why I thought **frameworks** is more suitable. I checked in Grammarly by pasting the paragraph and it is suggesting frameworks.